pages/common/vcsh.md

```diff
@@ -0,0 +1,21 @@
+# vcsh
+
+> Version Control System for $HOME - multiple Git repositories in $HOME.
+
+- Execute git command on managed repository:
+`vcsh {{repository_name}} {{git_command}}`
+
+- Initialize (empty) repository:
+`vcsh init {{repository_name}}`
+
+- Clone repository into custom name:
+`vcsh clone {{git_url}} {{repository_name}}`
+
+- List all managed repositories:
+`vcsh list`
+
+- Pull/pull all managed repositories from/to remotes:
```

Comment:
Perhaps `Push or pull` reads better.

Reply:
I think due to the sentence structure

Push or pull all managed repositories from or to remotes
Push/pull all managed repositories from/to remotes

the second one is more readable as the dashes indicate a selection better with the "from/to"; and I would prefer it over "from or to" - but I do not insist on it

Reply:
yes, i think the second one seems fine.